runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoConfig.java

```diff
@@ -52,4 +52,15 @@ default boolean showPercent()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "hideOnExisting",
+		name = "Hide on existing vanilla healthbar",
+		description = "NPC's that are known to have vanilla health bars will not have a Runelite one.",
```

Comment:
This description is really awkward. How about: "Hides Runelite health bar for NPCs who have a vanilla healthbar."

Reply:
Why even use the word vanilla? 

How about: "Hides Runelite health bar for NPCs who already have a healthbar visible in game."

Reply:
Its because everyone knows what it means and this way everyone will understand.